Plug 548

jyotibhalerao opened the pull request:
Code changes for task 548 Naming convention for Cxviewer project for VS2019 and VS2022 support.
Renamed projects for CxAction,CxViewerVSIX and shared projects.

jyotibhalerao committed:
Plug-548  Naming convention for CxViewer project for VS2019 and VS2022 support

Code changes for Plug-548 Naming convention for CxViewer project for VS2019 and VS2022 support.

jyotibhalerao committed:
Rename of CxActionShared project to CxViewerActionShared

Code changes for CxActionShared project to CxViewerActionShared

jyotibhalerao committed:
Integration branch for visual studio plugin changes

Code for features included below:
1)Authentication fix that impacted hosted customers
2)HTTPS - No fix as such as it works by adding certificate to the windows certificate store
3)Additional log messages
4)Fix for PLUG-513 - Results browsing and not being able to see CxPath, CxGraph
5)Not able to install VS2022 plugin on VS 2022 version lower than 17.0.1
6)Internal structuring change - Project names are changed
7)The flag that enabled lib64 zip
8)Support for complex solution (For ex. Open Web site code directly without a solution or scanning a folder of a solution)
9)Unbind so that if user open another solution closing the current one, bind the operation will work successfully
10)Plugin new icon and documentation link

SubhadraSahoo reviewed (changes requested):
@jyotibhalerao , Please revert the files got checked in as part of release and object folder

jyotibhalerao committed:
Code review changes

Removed unwanted bin, obj folder and changed version from 9.0.10 to 9.0.12 in manifest file for VS19 and VS22.

jyotibhalerao committed:
Made VS2022 manifest prerequisite version to 17.0

Made VS2022 manifest prerequisite version to 17.0 from 17.1.32112

jyotibhalerao committed:
Added diagnostic logs Info() and grammatically corrected log statements

Added diagnostic logs Info() and grammatically corrected log statements.

jyotibhalerao committed:
Changed version of log4net dll from 1.2.10.0 to 2.0.14.0

Changed version of log4net dll from 1.2.10.0 to 2.0.14.0 and corrected 3 log statements grammatically.

jyotibhalerao committed:
Log4net library version changed  missed file

Log4net library version changed  missed file

jyotibhalerao committed:
Update CxViewerAction2022.csproj for log4net library version from 1.2.10.0 to 2.0.14.0

Update CxViewerAction2022.csproj for log4net library version from 1.2.10.0 to 2.0.14.0

jyotibhalerao committed:
Build 9.0.12 bug fixes

Build 9.0.12 bug fixes:
1)when  scan any folder and file retrieve results option not  coming.
2)For any new solution , all five options are getting displayed even before first scan. Only 'Scan' and 'Bind' should be displayed .Rest options should remain disabled.
3)1After logout from SAST server able to perform scan the solution of project. It does not ask to login again.

jyotibhalerao committed:
Updated Newtonsoft.json library version from 13.0.1 to 9.0.1

Updated Newtonsoft.json library version from 13.0.1 to 9.0.1 and add one property in Property group of ci.msbuild for single build and remove one log statement from LoginHelper.cs file
